src/css/reset.scss

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Lesson learned from last project: ADD THIS FIRST! A pain in the butt to have to adjust everything when adding a reset file at the end of a project.